consensus/safety-rules/src/lib.rs

```diff
@@ -3,7 +3,11 @@
 
 #![forbid(unsafe_code)]
 
+#[macro_use]
+extern crate lazy_static;
```

Comment:
I don't think this syntax is required any more for our current compiler
https://doc.rust-lang.org/edition-guide/rust-2018/macros/macro-changes.html

Reply:
Done.

My rust 2018 is very _rusty_